app/models/metric/long_term_averages.rb

```diff
@@ -62,7 +62,8 @@ def self.get_averages_over_time_period(obj, options = {})
       Metric::Aggregation::Process.average(c, nil, results[:avg], counts)
 
       begin
-        results[:dev][c]  = vals[c].stddev
+        results[:dev][c] = vals[c].length == 1 ? 0 : vals[c].stddev
```

Comment:
I was thinking about that, but then I ruled that out.

I liked the fact that `stddev` behaves consistently with sample standard deviation formula. I know very little about the purpose of more_core_extensions, however it felt there is an ambition to have that thing useful even outside of context MiQ. Some other use may want to get NaN explicitly.

Anyway just my $0.02.